components/Profile.jsx

```diff
@@ -1,7 +1,40 @@
 import React from "react";
 
-const Profile = () => {
-  return <div>Profile</div>;
+const Profile = ({ name, major, classOf, netId, email, points }) => {
+  return (
+    <div className="grid grid-cols-2 w-[450px] p-[142px]">
+      <div>
+        <h1 className="text-[36px] h-fit w-fit font-bold">name:</h1>
+        <h1 className="text-[30px] h-min w-[370px] pb-[32px] 30">{name}</h1>
+      </div>
+      <div className="pl-[300px]">
+        <h1 className="text-[36px] h-fit w-fit font-bold">netid:</h1>
+        <h1 className="text-[30px] h-min w-[370px] pb-[32px] 30">{netId}</h1>
+      </div>
+
+      <div>
+        <h1 className="text-[36px] h-fit w-fit font-bold">major:</h1>
+        <h1 className="text-[30px] h-min w-[370px] pb-[32px] 30">{major}</h1>
+      </div>
+      <div className="pl-[300px]">
+        <h1 className="text-[36px] h-fit w-fit font-bold">email:</h1>
+        <h1 className="text-[30px] h-min w-[370px] pb-[32px] 30">{email}</h1>
+      </div>
+
+      <div>
+        <h1 className="text-[36px] h-fit w-max font-bold">class of:</h1>
+        <h1 className="text-[30px] h-min w-[370px] pb-[32px] 30">{classOf}</h1>
+      </div>
+      <div className="pl-[300px]">
+        <h1 className="text-[36px] h-fit w-fit font-bold">points:</h1>
+        <h1 className="text-[30px] h-min w-[370px] pb-[32px] 30">{points}</h1>
+      </div>
+
+      <button class="h-[94px] w-[359px] px-6 m-2 text-[36px] font-bold text-acm-black transition-colors duration-150 bg-[#C693EA] rounded-lg focus:shadow-outline hover:bg-indigo-500">
+        edit profile
+      </button>
```

Comment:
possibly add a id to the button so we can easily target the buttons event in the future. I think currently the implementation of the button is fine since there is only one on the page that I can see but might still be good to have. 

I have not checked responsiveness on this e\component but see a lot of hard coded height and width values so we also got to make sure that it will react well with other screen sizes. Just something to keep in mind

Reply:
Thank you for all the feedback!